backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -1954,9 +1954,12 @@ void SurfaceSdlGraphicsManager::copyRectToOverlay(const void *buf, int pitch, in
 #pragma mark --- Mouse ---
 #pragma mark -
 
-void SurfaceSdlGraphicsManager::setMouseCursor(const void *buf, uint w, uint h, int hotspotX, int hotspotY, uint32 keyColor, bool dontScale, const Graphics::PixelFormat *format) {
+void SurfaceSdlGraphicsManager::setMouseCursor(const void *buf, uint w, uint h, int hotspotX, int hotspotY, uint32 keyColor, bool dontScale, const Graphics::PixelFormat *format, const byte *mask) {
 	bool formatChanged = false;
 
+	if (mask)
+		warning("SurfaceSdlGraphicsManager::setMouseCursor: Masks are not supported");
```

Comment:
This is a bit sad. Is there a way to support cursor masks in the SurfaceSDL backend?

Reply:
I will investigate.

Reply:
So, I think that will not be an easy addition because with 8-bit cursors it creates an 8-bit SDL surface for the cursor and calls SDL_BlitSurface to draw it, which supports color key but does not support masks.  It may be possible to detect an absent color and use that as color key as long as the cursor doesn't contain 256 unique colors AND transparent pixels (which is impossible with a MacOS cursor and highly unlikely with a Win cursor) but doing better than that will require a custom blit.